app/src/main/java/com/m/androidNativeApp/MainActivity.java

```diff
@@ -42,29 +44,35 @@ protected void onCreate(Bundle savedInstanceState) {
                 .builder()
                 .build();
 
-        client.query(tasksQuery).enqueue(new ApolloCall.Callback<AllTasksQuery.Data>() {
+        Client.client.query(tasksQuery).enqueue(new ApolloCall.Callback<AllTasksQuery.Data>() {
```

Comment:
why the double `Client`?

Reply:
I think I was trying to do something that made no sense. Thank you for helping me realize that it was wrong.